From: Debian QA Group <packages@qa.debian.org>
Date: Sat, 12 Nov 2016 01:16:08 +1100
Subject: 75_vtysh__vtysh.c__PAGER

---
 vtysh/vtysh.c | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/vtysh/vtysh.c b/vtysh/vtysh.c
index c491824..0e63d0d 100644
--- a/vtysh/vtysh.c
+++ b/vtysh/vtysh.c
@@ -250,10 +250,16 @@ vtysh_pager_init (void)
 
   pager_defined = getenv ("VTYSH_PAGER");
 
-  if (pager_defined)
+  if (pager_defined) {
     vtysh_pager_name = strdup (pager_defined);
-  else
-    vtysh_pager_name = strdup ("more");
+  } else {
+    struct stat pager_stat;
+    if (stat("/usr/bin/pager", &pager_stat) == 0) {
+      vtysh_pager_name = strdup ("/usr/bin/pager");
+    } else {
+      vtysh_pager_name = strdup ("more");
+    }
+  }
 }
 
 /* Command execution over the vty interface. */
